micronota/tests/test_workflow.py

```diff
@@ -1,114 +0,0 @@
-# ----------------------------------------------------------------------------
```

Comment:
That's not good - the whole testing pipeline for the workflow.py has been trashed ...